.github/workflows/unit-tests.yml

```diff
@@ -6,18 +6,27 @@ on:
   pull_request:
     paths-ignore:
       - '**.md'     
+
 jobs:
-  Ubuntu:
-    name: Run Ubuntu
-    runs-on: ubuntu-latest
+
+  build:
+    name: Build
+
+    strategy:
+      matrix:
+        runs-on: [ubuntu-latest, macOS-latest, windows-latest]
+      fail-fast: false
+
+    runs-on: ${{ matrix.runs-on }}
+
     steps:
     - name: Checkout
-      uses: actions/checkout@master
+      uses: actions/checkout@v1
 
-    - name: Set Node.js 10.x
+    - name: Set Node.js 12.x
```

Comment:
the runner carries node 12

Thankfully this also happens to fix the ENOMEM toolrunner test issues on Windows

Reply:
Creating actions docs recommends `  using: 'node12'`, do we officially support actions on node 10 and node 12? 
If so, I think it would be worth having the workflow matrix across node versions, perhaps with certain functions explicitly listed as not supported on node 10.

Reply:
we only use node12 to run actions. I think the runner initially carried node v10, and then switched to node v12 before GA